Tests/StreamChatUITests/AssertSnapshot.swift

```diff
@@ -5,6 +5,12 @@
 import SnapshotTesting
 import XCTest
 
+/// Whether or not to record all new references.
+var isRecording: Bool {
+    get { SnapshotTesting.isRecording }
+    set { SnapshotTesting.isRecording = newValue }
+}
+
```

Comment:
I don't like this tbh. This is dangerous since it can override all existing snapshots. This should be done with extreme caution and adding a convenient API is the exact opposite.

Reply:
This is part of the snapshot framework API, I just exposed it, so we don't need to import the framework everywhere. And yes it should be used carefully, but is very useful when writing new tests and we want to record all tests for that View Controller. But I'll remove it for extra safety then 👍